vllm/model_executor/layers/quantization/compressed_tensors/compressed_tensors.py

```diff
@@ -164,7 +164,8 @@ def _is_fp8_w8a8(self, weight_quant: BaseModel,
         is_symmetric_activation = input_quant.symmetric
         is_per_tensor_activation = (
             input_quant.strategy == QuantizationStrategy.TENSOR)
-        if not (is_symmetric_activation and is_per_tensor_activation):
+        if not (is_symmetric_activation  # noqa: SIM103
+                and is_per_tensor_activation):
```

Comment:
curious of the reason for this change

Reply:
https://docs.astral.sh/ruff/rules/needless-bool/

Basically `ruff` here tells you that you can just do 

```python
return not(is_symmetric_activation and is_per_tensor_activation)
```

but since we have a comment below, I think it's clearer if we just suppress this error and keep it as is